config/Cargo.toml

```diff
@@ -49,3 +51,6 @@ fuzzing = ["aptos-crypto/fuzzing", "aptos-types/fuzzing"]
 smoke-test = []
 testing = []
 tokio-console = []
+
+[package.metadata.cargo-machete]
+ignored = ["strum"]
```

Comment:
hmm, why do we need this?

Reply:
Strum is used for the enum iter implementation, but `cargo-machete` can't tell that (it thinks the dependency is not required). So, you need to tell `cargo-machete` to ignore the false positive 😄